content/docs/setup/kubernetes/minimal-install/index.md

```diff
@@ -69,7 +64,13 @@ to manage the lifecycle of Istio.
     $ helm init --service-account tiller
     {{< /text >}}
 
-1. Install Istio:
+1. Install the `istio-init` chart to bootstrap all the Istio's [Custom Resource Definitions or CRDs for short](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions):
+
+    {{< text bash >}}
+    $ helm install install/kubernetes/helm/istio-init --name istio-init --namespace istio-system
+    {{< /text >}}
+
```

Comment:
There should be a step about waiting for Istio's 58 crds to register.  On a new installation, I use this technique

```
kubectl get crds | wc -l
```
and if the value = 58, I know the CRD registration is done.  A better technique would be to check that all jobs created by istio-init are in completed 0/1 state.

Reply:
Done.